id-repository/id-repository-identity-service/Dockerfile

```diff
@@ -101,8 +96,4 @@ EXPOSE 8090
 
 ENTRYPOINT [ "./configure_start.sh" ]
 
-CMD wget -q --show-progress "${kernel_ref_idobjectvalidator_url}" -O "${loader_path_env}"/kernel-ref-idobjectvalidator.jar; \
```

Comment:
Please add kernel_ref_idobjectvalidator dependency in pom

Reply:
Kernel dependency removed from run time dependency added inside pom